zipline/gens/brokers/rh/iex.py

```diff
@@ -0,0 +1,234 @@
+import requests
```

Comment:
It is unclear to me why do we have IEX related code under robinhood. Are those two always used together?

Reply:
I decided to include IEX to pull in data since I couldn't get it using the Robinhood API. I couldn't get some of these things: last_trade_price, last_trade_size, last_trade_time, total_volume and vwap

I could leave it out, but I'll have an incomplete bar/tick.